lib/generator/exampleGenerator.ts

```diff
@@ -77,61 +113,137 @@ export default class Generator {
+            if (example.responses && example.responses[code]) {
+              this.translator.extractResponse(specItem,example.responses[code],code);
+            }
+          }
+        }
+        return true;
       }
-    }
-
-    const example = this.getExampleFromPayload(operationId, specItem);
-    if (!example) {
-      return [];
-    }
-    console.log(example);
+    });
+    this.payloadCache.mergeCache()
+  }
 
-    if (this.shouldMock) {
+  private async generateExample( operationId: string,
+    specItem: any, rule: ExampleRule) {
+    
+    this.translator.setRule(rule);
+    this.swaggerMocker.setRule(rule);
+    let example 
+    console.log(`start generated example for ${operationId}, rule:${rule.ruleName}`);
+    if (!this.shouldMock) {
+      example = this.getExampleFromPayload(operationId, specItem);
+      if (!example) {
+        return [];
+      }
+    } else {
+      example = {
+        parameters: {},
+        responses: this.extractResponse(specItem, {})
+      };
       this.swaggerMocker.mockForExample(
         example,
         specItem,
         this.spec,
         util.getBaseName(this.specFilePath).split(".")[0]
       );
-      console.log(example);
     }
 
+    log.info(example);
     const unifiedExample = this.unifyCommonProperty(example);
-
     const newSpec = util.referenceExmInSpec(
       this.specFilePath,
       specItem.path,
       specItem.methodName,
-      `${operationId}_Base`
+      `${operationId}_${rule.exampleNamePostfix}_Base`
     );
     util.updateExmAndSpecFile(
       unifiedExample,
       newSpec,
       this.specFilePath,
-      `${operationId}_Base.json`
+      `${operationId}_${rule.exampleNamePostfix}_Base.json`
     );
 
-    console.log(`start validating generated example for ${operationId}`);
+    log.info(`start validating generated example for ${operationId}`);
     const validateErrors = await validate.validateExamples(this.specFilePath, operationId, {
-      //   consoleLogLevel: "error",
+      //   consoleLogLevel: "error"
     });
-    // if (validateErrors.length > 0) {
-    //   console.error(validateErrors);
-    // }
+    if (validateErrors.length > 0) {
+      log.error(`the validation raised below error:`)
+      log.error(validateErrors);
+      return validateErrors;
+    }
+    console.log(`generated example for ${operationId}, rule:${rule.ruleName} successfully!`);
+    return []
+  }
 
-    return validateErrors;
+  public async generate(
+    operationId: string,
+    specItem?: any
+  ): Promise<readonly ModelValidationError[]> {
+    if (!this.spec) {
+      await this.load()
+    }
+    if (!specItem) {
+      specItem = this.getSpecItem(this.spec, operationId);
+      if (!specItem) {
+        console.error(`no specItem for the operation id ${operationId}`);
+        return [];
+      }
+    }
+    const ruleSet: RuleSet = []
+    ruleSet.push({
+      exampleNamePostfix: "Maximum",
+      ruleName: "Maximum"
```

Comment:
Can you change it to `MaximumSet` for better understanding.

Reply:
done